pan-domain-auth-play_2-4-0/src/main/scala/com/gu/pandomainauth/action/Actions.scala

```diff
@@ -119,17 +119,12 @@ trait AuthActions extends PanDomainAuth {
     GoogleAuth.validatedUserIdentity(token).map { claimedAuth =>
       val authedUserData = existingCookie match {
         case Some(c) => {
-          val existingAuth = try {
-            CookieUtils.parseCookieData(c.value, settings.publicKey)
```

Comment:
I think @DiegoVazquezNanini must have had a service that set the new format cookie to the old cookie name when he did this? That's a bit mad so removing it makes sense for the library as a whole but be aware @DiegoVazquezNanini this will probably be a breaking change.